test/csharp/codegen/BUILD

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//bazel:grpc_build_system.bzl", "grpc_sh_test")
+
+licenses(["notice"])
+
+grpc_sh_test(
+    name = "csharp_codegen_simple_test",
+    srcs = ["csharp_codegen_simple_test.sh"],
+    data = [
+        "//src/compiler:grpc_csharp_plugin",
+        "@com_google_protobuf//:protoc",
+    ] + glob(["simple/**"]),
+    uses_polling = False,
```

Comment:
Adding 
```
tags = [
        "no_windows",
],
```

will automatically skip the test on windows (and I think we want that for now).

Reply:
while at it, you should probably also add `noasan`, `notsan`, `noubsan`, `nomsan` since the sanitizers are failing on the CI and we don't need any of them (at least initially).